### DISCUSSION TITLE
#12. VATZ biweekly meeting at `2022-07-06 08:00:00 CT`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#159] [General]
![image](https://user-images.githubusercontent.com/6308023/176089371-1f71b474-3e94-4150-b529-7fc1ebcc2685.png)

> 3/15  (20% complete)

- Finished
   - #96 
   - #133 
   - #149 
 
 - Push to Sprint 12
    - #69 
    - #85 
    - #87
    - #88
       - #121 
       - #122 
       - #127 
       - #142 
       - #118 
    - #119 
    - #120 
    - #146 
    - #150
    - #151

# Progress Check
- https://github.com/dsrvlabs/vatz/issues/88

# Discussion 
- https://github.com/dsrvlabs/vatz/issues/157
- https://github.com/dsrvlabs/vatz/issues/146
- https://github.com/dsrvlabs/vatz/issues/163
- https://github.com/dsrvlabs/vatz/issues/166

**Comment by xellos00** [comment]
# 동요
- https://github.com/dsrvlabs/vatz/issues/88
- https://github.com/dsrvlabs/vatz/issues/119
- https://github.com/dsrvlabs/vatz/issues/142
- https://github.com/dsrvlabs/vatz/issues/150
- https://github.com/dsrvlabs/vatz/issues/151
- https://github.com/dsrvlabs/vatz/issues/164

**Comment by rootwarp** [comment]
# 준교

- https://github.com/dsrvlabs/vatz/issues/162
  - https://github.com/dsrvlabs/vatz-plugin-common/issues/1
  - https://github.com/dsrvlabs/vatz-plugin-common/issues/3
- Refactoring?

**Comment by kim201212** [comment]
# 형기
- #156 
   - https://github.com/dsrvlabs/vatz-plugin-persistence/pull/1

**Comment by meetrick** [comment]
# 황제
- https://github.com/dsrvlabs/vatz/issues/121
  - https://github.com/dsrvlabs/vatz-plugin-osmosis
- https://github.com/dsrvlabs/vatz/issues/152
   - https://github.com/dsrvlabs/vatz-plugin-relayer

**Reply by xellos00** [reply]
@meetrick 
I can't connect to https://github.com/dsrvlabs/vatz-plugin-osmosis, can you check its access status?
![image](https://user-images.githubusercontent.com/6308023/177471334-c8f31344-d2b5-4a08-8e17-e1ec3c2b1198.png)

**Reply by meetrick** [reply]
> @meetrick I can't connect to https://github.com/dsrvlabs/vatz-plugin-osmosis, can you check its access status? ![image](https://user-images.githubusercontent.com/6308023/177471334-c8f31344-d2b5-4a08-8e17-e1ec3c2b1198.png)

I modified collaborator member.

**Comment by heejin-github** [comment]
# 희진
- #118 
  - https://github.com/dsrvlabs/vatz-plugin-solana/pull/11
  - https://github.com/dsrvlabs/vatz-plugin-solana/pull/12
  - https://github.com/dsrvlabs/vatz-plugin-solana/pull/13

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/127
- https://github.com/dsrvlabs/vatz/issues/146
- https://github.com/dsrvlabs/vatz/issues/152
  - https://github.com/dsrvlabs/vatz-plugin-relayer/pull/1
  - https://github.com/dsrvlabs/vatz-plugin-relayer/pull/3
- https://github.com/dsrvlabs/vatz/issues/163

**Comment by xellos00** [comment]
- #152
   @rootwarp, @meetrick 
   Register issue if there's any further development need with relayers 
   
---
